pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -937,10 +937,7 @@ func verifyObjectSha(ctx *verifierContext, config *types.VerifyImageConfig,
 
 	imageHash := fmt.Sprintf("%x", imageHashB)
 	configuredHash := strings.ToLower(config.ImageSha256)
-	if configuredHash == "" {
-		log.Infof("no image hash provided, skipping root validation. Setting to %s for %s", imageHash, config.ImageID)
-		status.ImageSha256 = strings.ToUpper(imageHash)
```

Comment:
I think this code is there to handle uplinks when no sha is specified. I don't know if we have such images, or some customers using that.

Reply:
For VM images, SHA is a mandatory field while uplinking. And for container images, there is no SHA in config so it goes through resolver and makes sure SHA must be available when it comes to the verifier.

Reply:
OK, so then you're removal is fine.